hh/actions/university_data.yaml

```diff
@@ -887,210 +606,491 @@ nw:
+  - Johanesburg
   TSIBA Education NPC:
-  - Woodstock
+  - Newtown
+  Turaco Hospitality (Pty) Ltd:
+  - Randburg
+  Villioti Fashion Institute (Pty) Ltd:
+  - Hyde Park
+nc:
+  Sol Plaatje University (SPU):
+  - Kimberley
+  UNISA:
+  - Kimberley
+  Northern Cape Rural:
+  - Central Office
+  - De Aar
+  - Kuruman
+  - Kathu
+  - Upington
+  - Namaqualand
+  Northern Cape Urban:
+  - Central Office
+  - City Campus
+  - Moremogolo
+  - Phatsimang
+  Boston City Campus & Business College:
+  - Kimberley
+  Damelin:
+  - Nelspruit
+  Hugenote Kollege:
+  - Kimberely
+lp:
+  Tshwane University of Technology (TUT):
+  - Polokwane
+  UNISA:
+  - Giyani
+  - Makhado
+  - Polokwane
+  University of Limpopo (UL):
+  - Turfloop Campus
+  University of Venda (UNIVEN):
+  - Thohoyandou
+  Capricorn:
+  - Central Office
+  - Seshego
+  - Polokwane
+  - Ramokgopa
+  - Senwabarwana
+  Lephalale:
+  - Modimolle
+  - Central Office
+  - Lephalale
+  Letaba:
+  - Giyani
+  - Central Office
+  - Maake
+  - Tzaneen
+  Mopani:
+  - Phalaborwa
+  - Sir Val Duncan
+  - Central Office
+  Sekhukhune:
+  - Central Office
+  - CN Phatudi
+  - CS Barlow
+  - Apel
+  Vhembe:
+  - Thengwe
+  - Central Office
+  - Shingwedzi
+  - Mavhoi
+  - Makwarela
+  - Mashamba
+  - Tshisimani
+  - Musina
+  Waterberg:
+  - Central Office
+  - Thabazimbi
+  - Business
+  - Engineering
+  - IT&C
+  Boston City Campus & Business College:
+  - Tzaneen
+  - Polokwane
+  Christ Baptist Church Seminary NPC:
+  - Faunapark
+  CTU Training Solutions:
+  - Polokwane
+  Jeppe College:
+  - Polokwane
+  MANCOSA:
+  - Polokwane
+  Mediclinic (Limpopo Mediclinic Learning Centre):
+  - Polokwane
+  MSC Business College:
+  - Louis Trichardt
+  - Polokwane
+  Richfield Graduate School:
+  - Thohoyandou
+  - Mankweng
+  - Polokwane
+mp:
+  Tshwane University of Technology (TUT):
+  - Mbombela
+  - eMalahleni
+  UNISA:
+  - Middelburg
+  - Mbombela
+  University of Mpumalanga (UMP):
+  - Siyabuswa
+  - Mbombela
+  Vaal University of Technology (VUT):
+  - Secunda
+  Ehlanzeni:
+  - Central Office
+  - Barberton
+  - Mapulaneng
+  - Mlumati
+  - Lydenburg
+  - Kanyamazane
+  - Mthimba
+  - Nelspruit
+  Gert Sibande:
+  - Central Office
+  - Sibanesetfu
+  - Perdekop
+  - Standerton
+  - Ermelo
+  - Evander
+  - Balfour
+  Nkangala:
+  - Middelburg
+  - Central Office
+  - Witbank
+  - Waterval Boven
+  - CN Mahlangu
+  - Mpondozankomo
+  Boston City Campus & Business College:
+  - Mbombela
+  - Witbank
+  Centurion Akademie:
+  - Witbank
+  CTU Training Solutions:
+  - Mbombela
+  Health and Fitness Professionals Academy (HFPA):
+  - Mbombela
+  Jeppe College:
+  - Mbombela
+  MANCOSA:
+  - Mbombela
+  Mediclinic (Nelspruit Mediclinic Learning Centre):
+  - Mbombela
+  MSC Business College:
+  - Nelspruit
+  Mukhanyo Theological College NPC:
+  - KwaMhlanga
+  Pearson Instittute of Higher Education:
+  - Nelspruit
+  Richfield Graduate School:
+  - Middelburg
+  - Mbombela
+  - eMalahleni
```

Comment:
This is a lot of changes. Maybe we should just double check things before making these changes?